content/en/docs/concepts/workloads/autoscaling.md

```diff
@@ -79,7 +79,7 @@ Mode | Description
 
 #### Requirements for in-place resizing
 
-{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+{{< feature-state feature_gate_name="InPlacePodVerticalScaling" >}}
```

Comment:
We need to update the `/content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md` file as well.